content/hi/docs/reference/glossary/manifest.md

```diff
@@ -0,0 +1,15 @@
+---
+title: मैनिफेस्ट (Manifest)
+id: manifest
+date: 2019-06-28
+short_description: >
+   एक या एक से अधिक क्यूबर्नेट्स एपीआई ऑब्जेक्ट का एक अनुक्रमित विनिर्देश।
+
+aka:
+tags:
+- fundamental
+---
+क्यूबर्नेट्स एपीआई ऑब्जेक्ट का विनिर्देश [JSON](https://www.json.org/json-en.html) या [YAML](https://yaml.org/) फ़ॉर्मेट में।
+
+<!--more-->
+मैनिफेस्ट उस ऑब्जेक्ट की इच्छित स्थिति को निर्दिष्ट करता है जिसे क्यूबर्नेट्स, मैनिफेस्ट को लागू करने पर बनाए रखेगा। YAML फ़ॉर्मेट के लिए, प्रत्येक फ़ाइल में कई मैनिफेस्ट शामिल हो सकते हैं।
```

Comment:
```suggestion
मैनिफेस्ट उस ऑब्जेक्ट की इच्छित स्थिति को निर्दिष्ट करता है जिसे Kubernetes, मैनिफेस्ट को लागू करने पर बनाए रखेगा। YAML फ़ॉर्मेट के लिए, प्रत्येक कॉन्फ़िगरेशन फ़ाइल में कई मैनिफेस्ट शामिल हो सकते हैं।
```

Reply:
@niranjandarshann Thank you!! update the changes.